### PR TITLE
update etag in factory calendar tutorial

### PR DESCRIPTION
**Pull request by mervey45**

### CODE REVIEW

File: tutorials/abap-environment-branch/abap-environment-branch.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Create new branch)]
+  1. Switch to your ABAP system in your SAP Fiori launchpad, open your software component **`Z_SWCT_XXX`** in **Manage Software Component**.
+
+      ![Create new branch](branch5.png)
+
+  2. Refresh your page by pressing **`F5`**.
+
+      ![Create new branch](branch.png)
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-branch/abap-environment-branch.md]: Why do you need a picture here? You have already told the user to press F5.

Comment (mervey45) [reply tutorials/abap-environment-branch/abap-environment-branch.md]: Sometimes users don't read, then they skip this step and wonder why it's not working. From my previous tutorials, I have found that such images are useful. That's why I think, thank it is important.

**Comment by mervey45** [comment]
@MichaelCzcz Can you merge my last change? I need this change by tomorrow for a workshop. Thank you.

**Comment by mervey45** [comment]
@MichaelCzcz I updated some changes, can you please merge my request by tomorrow. It's urgent. Thank you.

Best regards, 
Merve

**Comment by MichaelCzcz** [comment]
We would like to make the picture slightly smaller, but we will merge the PR now.

**Pull request merged**